File: 01_Basico/02 - Controle_Selecao/Resposta_em_JavaScript/questao04.js
var horario = prompt("Em quam horário você estuda ?\n[M]- matutino\n[V] - vespertino\n[N] - noturno\nreposta: ")
switch(horario[0].toUpperCase()){
    case "M":
        console.log("Bom dia")
        break
    case "V":
        console.log("Boa tarde")
    case "N":
        console.log("Boa noite")
        break
    default:
        console.log("Opção inválida")
}

File: 01_Basico/02 - Controle_Selecao/Respsotas_em_Python/questao04.py
horario = input("Em quam horário você estuda ?\n[M]- matutino\n[V] - vespertino\n[N] - noturno")
# Em pytohn não temos o comando switchcase, porém da para resolver com if 
if horario[0].upper() == "M":
    print("Bom dia")
elif horario[0].upper() == "V":
    print("Bom tarde")
elif horario[0].upper() == "N":
    print("Boa noite")
else:
    print("Opção inválida")
